src/common/options.cc

```diff
@@ -7491,6 +7491,10 @@ std::vector<Option> get_mds_options() {
     .set_flag(Option::FLAG_NO_MON_UPDATE)
     .set_description("path to MDS data and keyring"),
 
+    Option("mds_fs", Option::TYPE_STR, Option::LEVEL_ADVANCED)
```

Comment:
I think this option name is not clear enough. What about `mds_affinity_fs`?

Reply:
How about `mds_join_fs` which is consistent with other names we use like the `joinable` fs flag.

Reply:
?

Reply:
ah, just saw this. works for me!